lms/djangoapps/grades/new/course_grade.py

```diff
@@ -60,22 +73,42 @@ def has_access_to_course(self):
         """
         return len(self.course_structure) > 0
 
-    @lazy
+    @property
+    def percent(self):
+        """
+        Returns a rounded percent from the averall grade.
+        """
+        grade_summary = self.grade_summary
+        return round(grade_summary['percent'] * 100 + 0.05) / 100
```

Comment:
Once again, no need for a temporary variable.  Can just use `self.grade_summary` on this line and delete the previous line.